edx_when/models.py

```diff
@@ -125,6 +138,15 @@ class UserDate(TimeStampedModel):
     actor = models.ForeignKey(
         get_user_model(), null=True, default=None, blank=True, related_name="actor", on_delete=models.CASCADE
     )
+    first_component_block_id = UsageKeyField(null=True, blank=True, max_length=255, db_index=True)
```

Comment:
The same as above about performance - could you please add some comments in the PR about the frequency of filtering by this field.